### PR TITLE
Ignore false positive secret flagged by GitGuardian

### PR DESCRIPTION
**Pull request by dzhavat**
## References

- #0000

### CODE REVIEW

File: .gitguardian.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+matches-ignore:
```
Comment (LayZeeDK) [inline .gitguardian.yaml:1]: suggestion: try renaming to `gitguardian.yml` as per [the documentation](https://docs.gitguardian.com/internal-repositories-monitoring/gg_shield/configuration#configuration-file).

Comment (dzhavat) [reply .gitguardian.yaml]: Spot on! I'm pretty sure I copied the file name from the documentation somewhere. They have a typo as well :) 

But that doesn't seem to have an effect as well. In fact, the GitGuardian link in the checks section points to the previous commit.

**Comment by LayZeeDK** [comment]
I tried squashing the commits.

File: .gitguardian.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+matches-ignore:
+  - name: nxCloudAccessToken
+    match: 994b8e7c00028ee5981c35896a36f38356a7997f # read-only token
```
Comment (dzhavat) [inline .gitguardian.yml:3]: Question: Where is this value coming from?

Comment (LayZeeDK) [reply .gitguardian.yml:3]: It's a commit hash. Trying to figure out what the correct value is.

**Pull request closed without merging**